frontend/pages/settings/personal-access-tokens/new.tsx

```diff
@@ -110,9 +99,26 @@ export default function NewToken() {
           />
         )
       }),
-    [selectedActions],
+    [selectedActions, TokenWriteAction, TokenReadAction],
   )
 
+  if (isUiConfigLoading) {
```

Comment:
I'd recommend moving the loading and error handlers to just above your main return block at the bottom. Error handling should also come before loading.
```
  if (isUiConfigError) {
    return <MessageAlert message={isUiConfigError.info.message}></MessageAlert>
  }
  
  if (isUiConfigLoading) {
    return <Loading />
  }
  
  return (
     ...
   )
```